src/textual/css/stylesheet.py

```diff
@@ -336,6 +337,7 @@ def parse(self) -> None:
                 raise
             if any(rule.errors for rule in css_rules):
                 error_renderable = StylesheetErrors(css_rules)
+                self._invalid_css.add(css)
```

Comment:
This was the root cause of CSS errors being repeating. Failures here were never recorded.